recipes/_common.rb

```diff
@@ -1,17 +1,12 @@
-
-case node[:platform_family]
-when %r(debian|ubuntu)
-  package 'selinux-utils'
-when 'rhel', 'fedora'
-  package 'libselinux-utils'
-else
-    # implement support for your platform here!
-    raise "#{node[:platform_family]} not supported!"
+case node['platform_family']
+when 'debian'
+  include_recipe 'apt'
```

Comment:
This shouldn't be handled in this cookbook. You can handle it in the testing run list in Test Kitchen and a base recipe in your own environment. We don't want to be including it in every cookbook though.